app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt

```diff
@@ -889,64 +1047,75 @@ class AppointmentRepositoryAndroidTest {
       )
     }
 
-    fun createPatient(fullName: String): PatientProfile {
-      return testData.patientProfile(generatePhoneNumber = true)
-          .let { patientProfile ->
-            patientProfile.copy(patient = patientProfile.patient.copy(fullName = fullName))
-          }
-    }
-
-    fun scheduleAppointment(patientProfile: PatientProfile): Single<Appointment> {
+    fun scheduleAppointment(
+        appointmentUuid: UUID,
+        patientProfile: PatientProfile
+    ): Single<Appointment> {
       return appointmentRepository.schedule(
           patientUuid = patientProfile.patient.uuid,
-          appointmentDate = LocalDate.now(clock).minus(2, ChronoUnit.DAYS),
+          appointmentUuid = appointmentUuid,
+          appointmentDate = LocalDate.parse("2017-12-30"),
           appointmentType = Manual,
-          currentFacility = facility)
+          currentFacility = facility
+      )
     }
 
-    val patient1 = createPatient("Patient with one latest updated retro-active BP")
-    val patient2 = createPatient("Patient with no retro-active BP")
+    // given
+    val now = Instant.now(clock)
```

Comment:
We are using `clock` in some places whereas `testClock` in others.
Another reason why we should only inject one type.

Reply:
Agreed, I'll make this change throughout the codebase.